DAS/das/V2_mqtt_playback.py

```diff
@@ -0,0 +1,120 @@
+import sys
+import argparse
+import paho.mqtt.client as mqtt
+
+from das.utils.das_data_generator import send_csv_data, send_fake_data
+
+# Arguments
+parser = argparse.ArgumentParser(
+    description="DAS MQTT python script",
+    add_help=True,
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+parser.add_argument(
+    "-t",
+    "--time",
+    action="store",
+    type=int,
+    default=1,
+    help="length of time to send data",
+)
+parser.add_argument(
+    "-r",
+    "--rate",
+    action="store",
+    type=float,
+    default=0.5,
+    help="rate of data in seconds",
+)
+parser.add_argument(
+    "--host",
+    action="store",
+    type=str,
+    default="localhost",
+    help="address of the MQTT broker",
+)
+parser.add_argument(
+    "--port", action="store", type=int, default=1883, help="port of the MQTT broker"
+)
+parser.add_argument(
+    "--username",
+    action="store",
+    type=str,
+    help="username for the MQTT broker (optional)",
+)
+parser.add_argument(
+    "--password",
+    action="store",
+    type=str,
+    help="password for the MQTT broker (optional)",
+)
+parser.add_argument(
+    "-f",
+    "--file",
+    action="store",
+    type=str,
+    help="The csv file to replay. If not specified, makes up data.",
+)
+parser.add_argument(
+    "-s",
+    "--speed",
+    action="store",
+    type=float,
+    default=1,
+    help="Replay speed multiplier",
+)
+parser.add_argument(
+    "-j",
+    "--jump",
+    action="store",
+    type=int,
+    default=0,
+    help="Starts replaying from a specified time (in seconds)",
+)
+
+
+def start_publishing(client, args):
+    print("start")
+    client.publish("power_model/start")
```

Comment:
I think @hallgchris has a PR to add these topics to the common repo, so maybe we should wait for that PR to be merged in and then change these to use the topics directly from `mhp.topics`. It may bee annoying to have another PR to just change this later on.

Reply:
Nah I reckon it's fine. Whichever PR is ready second updates any changes from master. It shouldn't be too great of a change to make!

Reply:
I think that it should be easy to fix in another PR.